src/psyclone/tests/psyGen_test.py

```diff
@@ -2565,12 +2568,21 @@ def test_codeblock_view(capsys):
 
 
 def test_codeblock_can_be_printed():
-    '''Test that an CodeBlck instance can always be printed (i.e. is
+    '''Test that a CodeBlock instance can always be printed (i.e. is
     initialised fully)'''
     cblock = CodeBlock([])
     assert "CodeBlock[" in str(cblock)
     assert "]" in str(cblock)
 
+
+def test_codeblock_gen_c_code():
+    '''Test that a CodeBlock node fails to generate c code with a
+    GenerationError'''
+    cblock = CodeBlock([])
+    with pytest.raises(GenerationError) as err:
+        cblock.gen_c_code()
+    assert("CodeBlock can not be translated to C" in str(err.value))
```

Comment:
pylint complains that the parentheses are unnecessary.

Reply:
fixed